.env

```diff
@@ -0,0 +1,8 @@
+MYSQLDB_ROOT_PASSWORD=rfrfirf
```

Comment:
.env file shouldn't be stored in git
remove it and create .env.sample without values

Reply:
fix comment